build.sbt

```diff
@@ -23,12 +23,13 @@ lazy val microservice = Project(appName, file("."))
   .settings(majorVersion := 1)
   .settings(PlayKeys.playDefaultPort := 9702)
   .settings(
+      scalacOptions.+=("-Wconf:src=html/.*:s"),
       scalacOptions += "-Wconf:src=routes/.*:s",
-      scalacOptions += "-Wconf:cat=unused-imports&src=html/.*:s"
+      scalacOptions += "-Wconf:msg=Flag.*repeatedly:s"
   )
   .settings(
     onLoadMessage := "",
-    scalaVersion := "2.13.16",
+    scalaVersion := "3.7.0",
```

Comment:
3.7.x is not on LTS, is there any reason we want to have this version of scala upgrade. (3.3.x version is on LTS)

Reply:
As mentioned on standup, lets not get fixated on LTS here. Scala follows Rust model where the latest version can be used. And, per recommendations from there for a standard application, which this is, recommendation is to use latest version. LTS on scala is mainly for published libraries.

So, if there is a need for us to bump the version beyond LTS for a specific reason, we shouldn't hesitate to. As was needed in some of the services due to dependencies only been available for scala versions 3.5.x and beyond.

cc: @ruthdas03 @gajalakshmiseetharaman